src/System.Private.ServiceModel/tools/scripts/WCFCustomCleanup.cmd

```diff
@@ -0,0 +1,36 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
You will be able to do the same, just replace StartBridge with StartWCFSelfHostedSvc.cmd, will will call StartWCFSelfHostedSvcDoWork.cmd in another window, so it does not steal your window.